nixos/modules/services/networking/openvpn.nix

```diff
@@ -223,7 +223,7 @@ in
 
   config = mkIf (cfg.servers != { }) {
 
-    systemd.services = (listToAttrs (mapAttrsFlatten (name: value: nameValuePair "openvpn-${name}" (makeOpenVPNJob value name)) cfg.servers))
+    systemd.services = (listToAttrs (mapAttrsToList (name: value: nameValuePair "openvpn-${name}" (makeOpenVPNJob value name)) cfg.servers))
```

Comment:
Pulled in because of a top-level `with` statement 😿

Reply:
Welp 🤷
See also https://github.com/NixOS/nixpkgs/issues/208242

Reply:
See also who it's assigned to. 😉